idunn/api/places.py

```diff
@@ -60,3 +64,14 @@ def get_place_latlon(lat: float, lon: float, es: Elasticsearch, lang=None, verbo
         closest_place = None
     place = Latlon(lat, lon, closest_address=closest_place)
     return place.load_place(lang, verbosity)
+
+
+def handle_option(id, es: Elasticsearch, indices: IndexNames, lang=None, type=None, verbosity=DEFAULT_VERBOSITY):
+    if settings.get('ENABLE_OPTION_REQUESTS', False) is True:
+        headers = {
+            'Access-Control-Allow-Origin': '*',
+            'Access-Control-Allow-Headers': '*',
```

Comment:
It's unclear if this wildcard value is well supported by browsers: https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Access-Control-Allow-Headers#Compatibility_notes

Maybe it would be safer to use the value set in `Access-Control-Request-Headers` request header